registry/repository_test.go

```diff
@@ -531,6 +546,35 @@ func TestPutSignatureManifest(t *testing.T) {
 				annotations: map[string]string{
 					artifactspec.AnnotationArtifactCreated: validTimestamp,
 				},
+				signatureMediaType: joseTag,
+			},
+		},
+		{
+			name:      "succeed to put signature manifest with cose media type",
+			expectErr: false,
+			expectDes: notation.Descriptor{
+				MediaType: artifactspec.MediaTypeArtifactManifest,
+				Digest:    digest.Digest(validDigestWithAlgo10),
+				Size:      364,
+			},
+			expectManifest: SignatureManifest{
+				Annotations: map[string]string{
+					artifactspec.AnnotationArtifactCreated: validTimestamp,
+				},
+				Blob: notation.Descriptor{
+					MediaType: coseTag,
+					Digest:    validDigestWithAlgo5,
+				},
+			},
+			args: args{
+				reference:    validReference,
+				signature:    make([]byte, 0),
+				ctx:          context.Background(),
+				remoteClient: mockRemoteClient{},
+				annotations: map[string]string{
+					artifactspec.AnnotationArtifactCreated: validTimestamp,
+				},
+				signatureMediaType: coseTag,
```

Comment:
Why is cose test different from jws?

Reply:
What do you mean by different? This test is just pushing a signature with type `application/cose`.

Reply:
So in jws test we only have `expectDes` while in cose we have `expectDes`, `expectManifest` and `args`? Am I missing something?

Reply:
![image](https://user-images.githubusercontent.com/30316518/193202699-0b64cd0b-1df8-427f-adf1-a120e876bb69.png)
You can expand the line above, it is actually the same :)

Reply:
I remembered expanding the lines but somehow missed it :(